cmd/lncli/invoicesrpc_active.go

```diff
@@ -208,12 +211,14 @@ func addHoldInvoice(ctx *cli.Context) error {
 		return nil
 	}
 
-	hash, err := hex.DecodeString(args.First())
```

Comment:
Why not leave in place the old positional parsing as well? Otherwise, this may break any active scripts in the wild.

Reply:
Restored